sound_byte/lib/pages/friendScreen.dart

```diff
@@ -74,7 +77,12 @@ class _FriendScreenState extends State<FriendScreen> {
               Container(
                 child: TextField(
                   onChanged: (String input) {
-                    //TODO: update the list based on input
+                    Firestore.instance.collection('users').where("name", isEqualTo: input).getDocuments().then((querySnapshot)
```

Comment:
The firestore access is dangerous as id the searchedUser is used before the then executes it will error